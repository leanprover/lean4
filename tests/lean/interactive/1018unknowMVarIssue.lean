/-!
The expected type used to show "unknown metavariable '?_uniq.31828'" because of missing
backtracking of the info tree.
-/

inductive Fam2 : Type → Type → Type 1 where
  | any : Fam2 α α
  | nat : Nat → Fam2 Nat Nat

example (a : α) (x : Fam2 α β) : β :=
  match x with
  | Fam2.any   => _
                --^ $/lean/plainTermGoal
  | Fam2.nat n => n
