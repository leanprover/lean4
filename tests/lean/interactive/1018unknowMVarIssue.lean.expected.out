{"textDocument": {"uri": "file:///1018unknowMVarIssue.lean"},
 "position": {"line": 11, "character": 18}}
{"range":
 {"start": {"line": 11, "character": 18}, "end": {"line": 11, "character": 19}},
 "goal": "α✝ β : Type\nx : Fam2 α✝ β\nα : Type\na : α\n⊢ α"}
